src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +350,59 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info("Starting the agent services")
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
```

Comment:
please use `service_task` for task type.

Reply:
done